backend/substrapp/tasks/tasks.py

```diff
@@ -318,16 +323,17 @@ def prepare_materials(subtuple, tuple_type):
 
     # create subtuple
     subtuple_directory = build_subtuple_folders(subtuple)
+    put_algo(subtuple_directory, algo_content)
     put_opener(subtuple, subtuple_directory)
-    put_data_sample(subtuple, subtuple_directory)
     put_metric(subtuple_directory, metrics_content)
-    put_algo(subtuple_directory, algo_content)
+    put_data_sample(subtuple, subtuple_directory)
```

Comment:
Why do we have to change the order?
Does it matter?

Reply:
Nope, it found it better to order like this :)